packages/maskbook/src/plugins/Trader/settings.ts

```diff
@@ -97,13 +100,19 @@ const coinMarketCapPreferredCoinId = createInternalSettings<string>(
     `${PLUGIN_IDENTIFIER}+currentCoinMarketCapPreferredCoinId`,
     '{}',
 )
+const coinUniswapPreferredCoinId = createInternalSettings<string>(
```

Comment:
use table driven.

Reply:
It's a good chance to try table driven! But this pr is urgent, will do it further.